### ISSUE TITLE
Add example(s) that use the TensortRT-optimized SavedModels

### ISSUE THREAD

**Issue by RobertLucian** [cortexlabs/cortex#1061]
#### Description

Optimizing SavedModels with TensorRT can improve the model's performance substantially with minimal loss of accuracy:

* https://docs.nvidia.com/deeplearning/frameworks/tf-trt-user-guide/index.html
* https://github.com/tensorflow/tensorrt
* https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models

#### Motivation

Better showcase how far the models' performance can be taken. It wouldn't actually need the Triton Inference Server https://github.com/cortexlabs/cortex/issues/1016. This can be done with TF-serving.
